sdk/spring/azure-spring-boot-test-aad-b2c/src/test/java/com/azure/test/aad/b2c/utils/AADB2CTestUtils.java

```diff
@@ -2,13 +2,13 @@
 
 public class AADB2CTestUtils {
 
-    public static final String AAD_B2C_USER_EMAIL = "AAD_B2C_USER_EMAIL";
-    public static final String AAD_B2C_USER_PASSWORD = "AAD_B2C_USER_PASSWORD";
-    public static final String AAD_B2C_TENANT = "AAD_B2C_TENANT";
-    public static final String AAD_B2C_CLIENT_ID = "AAD_B2C_CLIENT_ID";
-    public static final String AAD_B2C_CLIENT_SECRET = "AAD_B2C_CLIENT_SECRET";
-    public static final String AAD_B2C_REPLY_URL = "AAD_B2C_REPLY_URL";
-    public static final String AAD_B2C_SIGN_UP_OR_SIGN_IN = "AAD_B2C_SIGN_UP_OR_SIGN_IN";
-    public static final String AAD_B2C_PROFILE_EDIT = "AAD_B2C_PROFILE_EDIT";
+    public static final String AAD_B2C_USER_EMAIL =  System.getenv("AAD_B2C_USER_EMAIL");
+    public static final String AAD_B2C_USER_PASSWORD = System.getenv("AAD_B2C_USER_PASSWORD");
+    public static final String AAD_B2C_TENANT = System.getenv("AAD_B2C_TENANT");
+    public static final String AAD_B2C_CLIENT_ID = System.getenv("AAD_B2C_CLIENT_ID");
+    public static final String AAD_B2C_CLIENT_SECRET = System.getenv("AAD_B2C_CLIENT_SECRET");
+    public static final String AAD_B2C_REPLY_URL = System.getenv("AAD_B2C_REPLY_URL");
+    public static final String AAD_B2C_SIGN_UP_OR_SIGN_IN = System.getenv("AAD_B2C_SIGN_UP_OR_SIGN_IN");
+    public static final String AAD_B2C_PROFILE_EDIT = System.getenv("AAD_B2C_PROFILE_EDIT");
```

Comment:
How about changing the class name to AADB2CEnvrionemt? We could do it in another PR.

Reply:
Okay